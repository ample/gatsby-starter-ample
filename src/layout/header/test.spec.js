import React from "react"
import renderer from "react-test-renderer"

import { component as Header, fixtures } from "."

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Header
          main_navigation={fixtures.default.main_navigation}
          top_navigation={fixtures.default.top_navigation}
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
