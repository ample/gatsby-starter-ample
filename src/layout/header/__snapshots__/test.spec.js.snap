// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="header"
>
  <div>
    <div
      className="logo_container"
    >
      <a
        className="logo"
        href="/"
      >
        <svg
          viewBox="0 0 292.642 340.529"
        >
          <g>
            <g>
              <path
                d="M183.017 315.94c-3.61-15.747 6.689-33.17 25.385-42.987 18.688-9.808 71.15-37.94 71.15-37.94l6.924 30.535c7.247 31.714-15.002 64.002-49.693 72.163-21.475 5.048-47.37 6.178-53.766-21.771zM111.27 323.753a91.566 91.566 0 005.675 16.776 157.222 157.222 0 014.916-305.155L275.019 0l17.623 76.35c-27.351-16.405-63.045-22.577-99.318-14.165-49.635 11.526-86.344 46.89-96.898 87.81a37.41 37.41 0 00-.275 17.44 38.338 38.338 0 0046 28.69c36.188-8.401 18.127-57.647 66.273-68.83 32.638-7.577 49.863 11.45 52.235 21.703l3.484 14.981-91.261 43.758a118.729 118.729 0 00-34.66 24.665c-23.498 24.543-34.659 58.132-26.953 91.351z"
                fill="#0065f2"
              />
            </g>
          </g>
        </svg>
      </a>
      <button
        className="menu_button"
        onClick={[Function]}
      >
        <span>
          Menu
        </span>
        <svg
          height={21}
          viewBox="0 0 448 512"
          width={26}
        >
          <path
            d="M442 114H6a6 6 0 01-6-6V84a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6zm0 160H6a6 6 0 01-6-6v-24a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6zm0 160H6a6 6 0 01-6-6v-24a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6z"
          />
        </svg>
      </button>
    </div>
    <div
      className="navigation_container"
    >
      <ul
        className="navigation top_navigation"
      >
        <li>
          <a
            href="#"
            onClick={[Function]}
            rel="noopener"
            target="_blank"
          >
            Duis
          </a>
        </li>
        <li>
          <a
            href="#"
            onClick={[Function]}
            rel="noopener"
            target="_blank"
          >
            Cupidatat
          </a>
        </li>
        <li>
          <a
            href="#"
            onClick={[Function]}
            rel="noopener"
            target="_blank"
          >
            Ullamco
          </a>
        </li>
        <li>
          <a
            href="#"
            onClick={[Function]}
            rel="noopener"
            target="_blank"
          >
            Labore
          </a>
        </li>
        <li>
          <a
            href="#"
            onClick={[Function]}
            rel="noopener"
            target="_blank"
          >
            Consectetur
          </a>
        </li>
        <li>
          <a
            href="#"
            onClick={[Function]}
            rel="noopener"
            target="_blank"
          >
            Pariatur
          </a>
        </li>
      </ul>
      <ul
        className="navigation main_navigation"
      >
        <li>
          <a
            href="#"
            onClick={[Function]}
            rel="noopener"
            target="_blank"
          >
            Reprehenderit
          </a>
        </li>
        <li>
          <a
            href="#"
            onClick={[Function]}
            rel="noopener"
            target="_blank"
          >
            Officia sint
          </a>
        </li>
        <li>
          <a
            href="#"
            onClick={[Function]}
            rel="noopener"
            target="_blank"
          >
            Veniam
          </a>
        </li>
        <li>
          <a
            href="#"
            onClick={[Function]}
            rel="noopener"
            target="_blank"
          >
            Consequat
          </a>
        </li>
        <li>
          <span
            className="dropdown"
            onClick={[Function]}
            onKeyPress={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="dropdown_trigger"
            >
              Suspendisse semper
              <svg
                viewBox="0 0 256 512"
              >
                <path
                  d="M119.5 326.9L3.5 209.1c-4.7-4.7-4.7-12.3 0-17l7.1-7.1c4.7-4.7 12.3-4.7 17 0L128 287.3l100.4-102.2c4.7-4.7 12.3-4.7 17 0l7.1 7.1c4.7 4.7 4.7 12.3 0 17L136.5 327c-4.7 4.6-12.3 4.6-17-.1z"
                />
              </svg>
            </div>
            <ul
              className="dropdown_menu"
            >
              <li>
                <a
                  className="link"
                  href="/"
                  onClick={[Function]}
                >
                  Vestibulum vitae
                </a>
              </li>
              <li>
                <a
                  className="link is_external"
                  href="#"
                  onClick={[Function]}
                  rel="noopener"
                  target="_blank"
                >
                  Pellentesque tempor venenatis
                </a>
              </li>
              <li>
                <a
                  className="link is_external"
                  href="#"
                  onClick={[Function]}
                  rel="noopener"
                  target="_blank"
                >
                  Fusce molestie
                </a>
              </li>
              <li>
                <a
                  className="link is_external"
                  href="#"
                  onClick={[Function]}
                  rel="noopener"
                  target="_blank"
                >
                  Etiam in lacinia dolor
                </a>
              </li>
            </ul>
          </span>
        </li>
        <li
          className="nav_button"
        >
          <a
            className="button button theme_default"
            href="#"
            onClick={[Function]}
            rel="noopener"
            target="_blank"
          >
            Quisque interdum
          </a>
        </li>
      </ul>
    </div>
  </div>
</header>
`;
